desktop/src/main/java/bisq/desktop/main/portfolio/pendingtrades/PendingTradesView.java

```diff
@@ -316,6 +316,19 @@ private void openChat(Trade trade) {
         if (chatPopupStage != null)
             chatPopupStage.close();
 
+        if (trade.getCommunicationMessages().isEmpty()) {
```

Comment:
This will make receivers of messages from peers not see the code of conduct message